src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -793,17 +793,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return GetSymbolInfoForNode(options, GetBoundNodeSummary(node), binderOpt:=Nothing)
         End Function
 
-        Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
-            Dim summary = GetBoundNodeSummary(node)
-            Dim result As BoundNode
-            Select Case options
-                Case GetOperationOptions.Highest
-                    result = summary.HighestBoundNode
-                Case GetOperationOptions.Parent
-                    result = summary.LowestBoundNodeOfSyntacticParent
-                Case Else
-                    result = summary.LowestBoundNode
-            End Select
+        Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, cancellationToken As CancellationToken) As IOperation
+            Dim bindingRoot = DirectCast(GetBindingRoot(node), VisualBasicSyntaxNode)
```

Comment:
>GetBindingRoot [](start = 41, length = 14)

Even though GetBindingRoot function has the same name as in C#, it's behavior is different. In particular, it always jumps out of a lambda, which is by design. As a result, we end up requesting much bigger tree than we could. This is not a stopper for this PR because I do not believe this is going to lead to an incorrect behavior, but we should follow up.